Add Webots Privacy Policy (#2223)
cyberbotics/webots: docs/guide/privacy-policy.md

## Privacy Policy

Cyberbotics respects your privacy and it is committed to protecting privacy and personal data.

> This Privacy Policy applies to the personal data of "Webots" software that may be collected through [https://cyberbotics.com](https://cyberbotics.com).

Reviewer: ```suggestion
This Privacy Policy applies to the personal data of the "Webots" software that may be collected through [https://cyberbotics.com](https://cyberbotics.com).
```

By accepting this Privacy Policy and using Webots you acknowledge and agree to the terms and conditions of this Policy.

### Personal Data Collected

Personal data may be collected by the Webots telemetry system if activated from the Webots preferences or at first launch.
The telemetry system will collect information about the system configuration, the IP address, the Webots settings and the Webots world name.

### Use of Personal Data

All the collected personal information are only used internally by Cyberbotics to improve the Webots software.

> After being anonymized, the collected personal information are published for on the telemetry page on [https://cyberbotics.com](https://cyberbotics.com).

Reviewer: ```suggestion
After being anonymized, the collected personal information are published on the telemetry page at [https://cyberbotics.com/telemetry](https://cyberbotics.com/telemetry).
The purpose of this publication is openness and transparency.
It allows anyone to consider this data to better contribute to the Webots software.
```

In all such cases the transfer of personal data will be in accordance with Cyberbotics’s Personal Data Protection Policy, and in compliance with the GDPR.

### Public Forum

Cyberbotics S.à.r.l may offer support on public forums.
Please note that information that you disclose in these public forums become public.
Use caution in disclosing any personal information.
We may store the published messages in order to extend our software documentation and help other users.

### Retention of Personal Data

Cyberbotics S.à.r.l retains your personal information only for as long as necessary to comply with our legal obligations, resolve disputes, and enforce our agreements; and for so long as is necessary for the purposes for which we collected such personal information.

### Rights of the Users

You have the right, at any time, to request access, rectification and erasure of your personal information.
In addition, you have the right to withdraw your consent, without this affecting the lawfulness of processing based on consent before its withdrawal.

> To exercise your rights under these provisions, please contact us at the “Communications” details below.

Reviewer: ```suggestion
To exercise your rights under these provisions, please contact us (see “Communications” section below).
```

### Security of Personal Data

Cyberbotics fully understands the importance of privacy and takes all the necessary steps to ensure that all the collected personal data is processed, stored and deleted in a safe and secure way.

### Changes to This Privacy Policy

Cyberbotics reserves the right to make changes to this privacy policy at any time by given notice to its Users on this page.
It is strongly recommended to check this page often, referring to the date of the last modification listed at the bottom.
If a User objects to any of the changes to the Policy, the User may disable the telemetry system and can request Cyberbotics to erase the personal data.
Unless stated otherwise, the then-current privacy policy applies to all personal data Cyberbotics has about Users.

### Communications

If you have questions regarding this Privacy Policy, please contact us at [info@cyberbotics.com](mailto:info@cyberbotics.com).
<br/>

> Latest update: September 10, 2020.

Reviewer: ```suggestion
Latest update: September 14th, 2020.
```
